skimage/io/_plugins/matplotlib_plugin.py

```diff
@@ -169,10 +169,33 @@ def imshow_collection(ic, *args, **kwargs):
     """Display all images in the collection.
 
     """
-    fig, axes = plt.subplots(1, len(ic))
+    N = len(ic)
+    print N
```

Comment:
Oh, my mistake. I completely forgot that I had put a print statement there. I will also update to python3.